orange-datamodel/src/test/java/com/hartwig/hmftools/datamodel/OrangeJsonTest.java

```diff
@@ -311,6 +312,7 @@ private static void assertLinx(@NotNull LinxRecord linx)
         LinxFusion fusion = linx.allSomaticFusions().iterator().next();
         assertTrue(fusion.reported());
         assertEquals(LinxFusionType.KNOWN_PAIR, fusion.reportedType());
+        assertEquals(LinxUnreportableReason.NONE, fusion.unreportedReasons().iterator().next());
```

Comment:
You could consider adding a check for size == 1 since you'd expect 1 entry in the list of unreported reasons, and in current situation there could be more than 1